gen_import_utils.py

```diff
@@ -96,4 +97,20 @@ def cont_prompter():
         elif user_input.lower() == "n":
             sys.exit("Script terminated by user.")
         else:
-            print("Invalid input. Please enter 'y' or 'n'.")
\ No newline at end of file
+            print("Invalid input. Please enter 'y' or 'n'.")
+
+def generate_token(timestamp, filename):
```

Comment:
moved this function from sql_csv_utils to gen_import_utils, so that it can be used anywhere to generate md5s without a db_connection config.